src/main/java/com/aang23/globaltab/GlobalTab.java

```diff
@@ -131,4 +139,21 @@ public static void insertIntoTabListCleanly(TabList list, TabListEntry entry, Li
                 toKeep.add(inUUID);
         }
     }
+    public void schedule(){
+        this.task = server.getScheduler().buildTask(this, () -> {
+            // logger.info("Started AutoReload task!");
+
+            Timer timer = new Timer();
+            timer.scheduleAtFixedRate(new TimerHandler(),
+                    Integer.parseInt((String) ConfigManager.config.get("updatedelay")) * 1000,
+                    Integer.parseInt((String) ConfigManager.config.get("updatedelay")) * 1000);
+
+
+        }).repeat(10, TimeUnit.MINUTES).schedule();
```

Comment:
This can be configured however you'd like, can even put it into the config so you can allow server owners to set how often they want it to be scheduled.